src/util/filter-toolbox.js

```diff
@@ -39,6 +39,7 @@ var filterToolbox = function (toolbox, opcodes) {
             if (category.nodeName.toLowerCase() !== 'category') continue;
             var filteredCategory = filterToolboxNode(category, opcodes);
             if (filteredCategory.hasChildNodes()) filteredToolbox.appendChild(filteredCategory);
+            if (!filteredCategory.hasChildNodes() && filteredCategory.hasAttribute("custom")) filteredToolbox.appendChild(filteredCategory);
```

Comment:
When you fix the lint errors, would you please combine the line with the one before? I.e.,
```javascript
if (filteredCategory.hasChildNodes() ||
    filteredCategory.hasAttribute('custom')) {
    ...
}
```